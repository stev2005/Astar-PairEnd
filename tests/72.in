ATCCCGTACTTCCAAAACCCAAGGGAGCTAGATTCCCCCGAATCCACCCTGCTATCACGACAACACGCGTAGTTTATCCCAAGTAGACTGCCTCGTGAACTATGATTTAA
15
AATTCCTGACTCAATTGTATGATCAGAAGAAGCTCGAAACCTGGTGCC
CCAGACGGCGACTGGGTATTA
ATGGGAAAGCCGTCTGTAATCTCAGAACTCCTATGTAGTGTAATCTCATCTA
AATACATGCGGAACAGAGGAGCCCGCATACAACTAGTTAAGTCGACCCA
CGATAGAACCCGCTCGCCCGCTCGGCATGAGAGGTGATTCAATGTCACGCTTCAGTCGGAGAAAGTTGGGTTGTC
ACTGGGCTGTGTCATTATAGGTACCAATTCCTGACAGGAAGGATTTTTTTGCGGGTGGGCTAACACGTAGTGATCTTA
GTCTCTGAGATTATACTCTGCCGTGCCCACAGCCGGCATTAGGACGGAACTCTCCCGGGTTTCAATTCTGAAATACCGCCAATTCAA
G
TCCAGG
CTTTATTTAAAGCGGGCCTC
TTAGACTGTTTTGGTGCGGCGATATCCAAGGACGGGTCATAAGTGCCATTCCTAGGCTTGG
GTTACGGGCGGTC
AGATCGCCGTTAGGGATAAGACCGTGCCGGGGCGGGATTTGTTACCCCCCTCGAAGTCTCTCTAACTAAGTG
GTTTAAACTGTAGCTTCAGGTTGCGTCAGAGCTCATCCAATTACCTAGTGATCTATGCAACGCATGTATTAGTCTAATGACGTTC
GTAGAGACGTTGTGCAAACTACGGCACATGT
