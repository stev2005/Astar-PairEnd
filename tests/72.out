22
8
23
21
35
38
40
0
2
9
31
6
35
41
13
